pallets/capacity/src/lib.rs

```diff
@@ -407,18 +408,19 @@ pub mod pallet {
 		CannotChangeStakingType,
 		/// The Era specified is too far in the past or is in the future (15)
 		EraOutOfRange,
-		/// Rewards were already paid out for the specified Era range
-		IneligibleForPayoutInEraRange,
```

Comment:
Normally, that is true, but remember this particular error is not in production/main branch; otherwise I would have just marked it as deprecated.  However, in comparing it to main, I realized that "NoThawedTokenAvailable" somehow got moved to the bottom, and that is definitely not backward compatible.  So I am moving it now.

Reply:
Also thanks for flagging this - just to be sure, I just double checked the other error enums which I did rename, I think they all essentially retain the same meaning and are in the same order.  Let me know if you disagree.